### PR TITLE
test: Also refresh private images

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: test/testinfra.py [hunk 1]
```diff
@@ -90,6 +90,34 @@
     }
 }
 
+NON_PUBLIC_IMAGE_REFRESH = {
```
Comment (stefwalter) [inline test/testinfra.py]: Should this be called 'PRIVATE_IMAGE_REFRESH' instead?

File: test/testinfra.py [hunk 2]
```diff
@@ -90,6 +90,34 @@
...
+        ]
+    },
+    'rhel-atomic': {
+        'triggers': [ "verify/rhel-atomic" ]
+    }
+}
+
+## Add non-public images if we have a store for them
+
+NON_PUBLIC_STORE = "~/.config/non-public-image-store"
```
Comment (stefwalter) [inline test/testinfra.py]: PRIVATE_STORE?

File: test/testinfra.py (hunk 1 above)
Comment (mvollmer) [inline test/testinfra.py]: Heh, of course.

Comment (mvollmer) [inline test/testinfra.py]: Done.

File: test/testinfra.py (hunk 2 above)
Comment (mvollmer) [inline test/testinfra.py]: Done.

**Pull request closed without merging**